x-pack/docs/en/rest-api/security/create-roles.asciidoc

```diff
@@ -24,9 +24,12 @@ privilege.
 [[security-api-put-role-desc]]
 ==== {api-description-title}
 
-The role API is generally the preferred way to manage roles, rather than using
-file-based role management. For more information about the native realm, see 
-{stack-ov}/realms.html[Realms] and <<configuring-native-realm>>. 
+The role management APIs are generally the preferred way to manage roles, rather than using
+{stack-ov}/defining-roles.html#roles-management-file[file-based role management]. The create
+or update roles API cannot update roles that are defined in roles files.
+
+For more information about the native realm, see
+{stack-ov}/realms.html[Realms] and <<configuring-native-realm>>.
```

Comment:
Let's drop this sentence - index based roles aren't really related to the native realm.